api/api_controller_test.go

```diff
@@ -1444,72 +1444,97 @@ func TestHandler_ContinuousExportHandlers(t *testing.T) {
 }
 
 func Test_setupLakeFSHandler(t *testing.T) {
-	// get handler with DB without apply the DDL
-	handler, deps := getHandler(t, "", testutil.WithGetDBApplyDDL(false))
-
-	srv := httptest.NewServer(handler)
-	defer srv.Close()
-
 	name := "admin"
-	user := models.Setup{
-		Username: &name,
-	}
-	req, err := json.Marshal(user)
-	if err != nil {
-		t.Fatal("JSON marshal request", err)
+	cases := []struct {
```

Comment:
Consider adding tests for edge cases: I'm mostly worried about empty strings here.

Reply:
Done!